index.js

```diff
@@ -56,7 +56,7 @@ app.use(bodyParser.urlencoded({ extended: true }));
 // ==================================================================
 
 if (process.env.TWILIO_AUTH_TOKEN) {
-  app.post(
+  /*app.post(
```

Comment:
I think we can delete this twilio section depending on the answer to the package.json comment.

if we do delete it, maybe just make a note here that twilio is deployed separately and so written as a subpackage

Reply:
looks like @jorganisak also took this out in his PR

Reply:
alright, deleted!